docs/fundamentals/flow/index.md

```diff
@@ -7,13 +7,17 @@ Documents "flow" through the created pipeline and are processed by Executors.
-|----------------|----------------------------------------------------------------------------------------------------------------------------------------------|
-|  `.add()` | Add an Executor to the `Flow`                                                                                                                |
-| `with` context manager       | You can use the `Flow` as a context manager. It will automatically start and close your `Flow` then.                                         |
-| `.plot()` | Visualizes the flow. Helpful for building complex pipelines.                                                                                 |
-| `.post()`   | Sends requests to the API of the `Flow`.                                                                                                     |
-| `.block()`        | Blocks execution until the program is terminated. This is useful to keep the `Flow` alive so it can be used from other places (Clients etc). |
+| Method                             | Description                                                                                                                                  |
+|------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| `.add()`                           | Add an Executor to the `Flow`                                                                                                                |
+| `.start()`                         | Starts the `Flow`. This will start all its Executors and check if they are ready to be used.                                                 |
+| `.stop()`                          | Stops the `Flow`. This will stop all its Executors.                                                                                          |
+| `with` context manager             | You can use the `Flow` as a context manager. It will automatically start and close your `Flow` then.                                         |
+| `.to_docker_compose()`             | Generates a Docker-Compose file listing all its Executors as Services.                                                                       |
+| `.to_k8s_yaml(<output_directory>)` | Generates the Kubernetes configuration files into <output_directory>.                                                                        |
+| `.plot()`                          | Visualizes the flow. Helpful for building complex pipelines.                                                                                 |
+| `.post()`                          | Sends requests to the API of the `Flow`.                                                                                                     |
+| `.block()`                         | Blocks execution until the program is terminated. This is useful to keep the `Flow` alive so it can be used from other places (Clients etc). |
```

Comment:
```suggestion
| `.block()`                         | Blocks execution until the program is terminated. This is useful to keep the Flow alive so it can be used from other places (clients, etc). |
```